qiskit/circuit/quantumcircuit.py

```diff
@@ -2265,17 +2265,98 @@ def reset(self, qubit: QubitSpecifier) -> InstructionSet:
         return self.append(Reset(), [qubit], [])
 
     def measure(self, qubit: QubitSpecifier, cbit: ClbitSpecifier) -> InstructionSet:
-        """Measure quantum bit into classical bit (tuples).
+        """Measure quantum bit (qubit) into classical bit (cbit).
+
+        When a qubit is measured, its state collapses to a classical bit and copied to a
+        classical wire.
```

Comment:
This isn't strictly physically correct, and it's missing a bit of information.  I don't know how you want to present it to less informed users, but for knowledgable ones, we should mention that `measure` is specifically a projective measurement in the computational (Pauli Z) basis, and the classical bit indicates whether the result is the $\lvert1\rangle$ state (1) or the $\lvert0\rangle$ state (0).

It feels a bit off to say a quantum state "collapses to a classical bit" - the global wavefunction collapses such that the measured qubit is in a single eigenstate of the projective measurement. In a general entangled state, it's also not super precise to imply that a single qubit can have a state independent of the others.  I think I'd avoid talking about the quantum state too much here - if a user doesn't already know what a "projective measurement" is, they'll be better served by an intro to QC rather than us trying to fit a miniature version into API documentation.

Reply:
What do you think about the explanation in e3bc46abfe6bfccc9fe5b6238c9d00048e554d59 . It would be great to keep it simple while technically correct.